docs/axes/cartesian/README.md

```diff
@@ -31,6 +31,7 @@ The following options are common to all cartesian axes but do not apply to other
 | `maxRotation` | `Number` | `90` | Maximum rotation for tick labels when rotating to condense labels. Note: Rotation doesn't occur until necessary. *Note: Only applicable to horizontal scales.*
 | `minRotation` | `Number` | `0` | Minimum rotation for tick labels. *Note: Only applicable to horizontal scales.*
 | `mirror` | `Boolean` | `false` | Flips tick labels around axis, displaying the labels inside the chart instead of outside. *Note: Only applicable to vertical scales.*
+| `offset` | `Boolean` | `false` | If true, extra space is added to the both edges and the axis is scaled to fit into the chart area. This is set to `true` in the bar chart by default.
```

Comment:
Sounds quite similar to the `scale.bounds` feature that was implemented in https://github.com/chartjs/Chart.js/pull/4556 (docs are being added in https://github.com/chartjs/Chart.js/pull/4592). Is it possible to use that option? I'm not quite sure if this is doing the same thing or not